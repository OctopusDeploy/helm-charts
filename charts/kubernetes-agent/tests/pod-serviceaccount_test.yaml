suite: "pod permissions"
templates:
- templates/pod-serviceaccount.yaml
tests:
- it: "should match snapshot"
  asserts:
  - matchSnapshot: {}