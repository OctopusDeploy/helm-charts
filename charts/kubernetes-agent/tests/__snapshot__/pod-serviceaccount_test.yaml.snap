should match snapshot:
  1: |
    apiVersion: v1
    automountServiceAccountToken: true
    kind: ServiceAccount
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: octopus-agent
        app.kubernetes.io/version: 8.2.2514
        helm.sh/chart: kubernetes-agent-1.19.1
      name: octopus-agent-scripts
      namespace: NAMESPACE
